plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveWriteUtils.java

```diff
@@ -723,4 +728,25 @@ public static HiveDecimal getHiveDecimal(DecimalType decimalType, Block block, i
         }
         return HiveDecimal.create(unscaledValue, decimalType.getScale());
     }
+
+    private static Timestamp getHiveTimestamp(TimestampType type, Block block, int position)
+    {
+        verify(type.getPrecision() <= MAX_PRECISION.getPrecision(), "Timestamp precision too high for Hive");
```

Comment:
We have so many definitions of `MAX_PRECISION` constant. Please qualify use with the class name

Reply:
Heh, I actually had it as `HiveTimestampPrecision.MAX` at first before deciding on this; I'll switch it back.